### DISCUSSION TITLE
How to fix Errors in war3sourcelog

### DISCUSSION THREAD

**Discussion by trollollol123** [War3Evo/War3Source-EVO#29] [Q&A] [answered]
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "fireorb_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "posthaste_diamondcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "dielaughing_diamondcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "scrollessence_diamondcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "fbracer_diamondcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "lifetube_diamondcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "sbracelt_diamondcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "trinket_diamondcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "uberme_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "tome_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "gauntlet_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "boot_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "ring_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "cloak_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "sock_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "antiward_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "mask_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "armband_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "claw_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "orb_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "fireorb_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "piercing_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "oil_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "plate_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "hope_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "shield_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "courage_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "lace_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "helm_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "fireorb_itemflags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "courage_itemflags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "uberme_itemflags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "armband_itemflags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "oil_itemflags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "piercing_itemflags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "plate_itemflags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "farseer_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "frogger_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "hyperC_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "heatseeker_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "flamepredator_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "bloodlord_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "builder_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "builder_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "mcyborg_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "scyborg_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "cyborgspy_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "dontplaythis_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "deathknight_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "dread_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "shapeshifter2_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "clinkz_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "farseer_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "deathknight_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "shapeshifter2_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "frogger_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "hyperC_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "heatseeker_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "flamepredator_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "bloodlord_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "scyborg_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "cyborgspy_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "mcyborg_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "soulmedic_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "applejack_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "nightguard_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "angel_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "clinkz_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "claw_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "fireorb_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "orb_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "mask_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "oil_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "piercing_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "armband_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "antiward_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "courage_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "plate_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "helm_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "hope_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "lace_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "shield_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "boot_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "cloak_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "ring_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "sock_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "gauntlet_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "tome_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "uber50_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "stophack_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "mboots_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "mring_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "mhealth_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "cyborgheavy_team1_limit" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "cyborgheavy_team2_limit" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "scout_o_restrictclass" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "link_restrictclass" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "smashbro_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "transorcerer_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "transorcerer_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "naix_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "dontplaythis_flags" not found, please fix/clean up your config
Wed Jun 29 19:30:54 2016 W3CVAR (internal)  "dontplaythis_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "fireorb_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "posthaste_diamondcost" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "dielaughing_diamondcost" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "scrollessence_diamondcost" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "fbracer_diamondcost" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "lifetube_diamondcost" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "sbracelt_diamondcost" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "trinket_diamondcost" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "uberme_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "fireorb_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "hope_goldcost" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "fireorb_itemflags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "uberme_itemflags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "farseer_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "frogger_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "hyperC_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "heatseeker_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "flamepredator_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "bloodlord_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "builder_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "builder_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "mcyborg_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "scyborg_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "cyborgspy_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "dontplaythis_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "deathknight_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "dread_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "shapeshifter2_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "clinkz_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "farseer_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "deathknight_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "shapeshifter2_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "frogger_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "hyperC_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "heatseeker_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "flamepredator_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "bloodlord_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "scyborg_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "cyborgspy_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "mcyborg_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "soulmedic_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "applejack_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "nightguard_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "angel_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "clinkz_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "fireorb_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "hope_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "uber50_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "stophack_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "mboots_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "mring_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "mhealth_itemcategory" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "cyborgheavy_team1_limit" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "cyborgheavy_team2_limit" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "scout_o_restrictclass" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "link_restrictclass" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "smashbro_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "transorcerer_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "transorcerer_accessflag" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "naix_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "dontplaythis_flags" not found, please fix/clean up your config
Wed Jun 29 19:46:53 2016 W3CVAR (internal)  "dontplaythis_accessflag" not found, please fix/clean up your config

**Comment by War3Evo** [comment, marked as answer]
Those can be fixed by removing them from the https://github.com/War3Evo/War3Source-EVO/blob/master/cfg/war3sourceALL.cfg file.

For now I am keeping them until I can fully update and upload those races and addons.
